src/keyring/KeyringController.test.ts

```diff
@@ -449,8 +563,42 @@ describe('KeyringController', () => {
     } catch (e) {
       error2 = e;
     }
-    expect(error1.message).toContain('Keyring Controller signTypedMessage:');
-    expect(error2.message).toContain('Keyring Controller signTypedMessage:');
+
+    expect((error1 as ErrorMessageObject).message).toContain(
```

Comment:
Instead of checking for the error like this, why not say:

``` javascript
expect(
  keyringController.signTypedMessage(
    { data: msgParams, from: account },
    SignTypedDataVersion.V1,
  )
).rejects.toThrow(/^Keyring Controller signTypedMessage:/u)
```

This way you don't have to typecast the error.

Reply:
Fixed